specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/apimpolicyrestrictions.json

```diff
@@ -0,0 +1,374 @@
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "200": {
+            "description": "Policy restriction configuration of the Api Management service was successfully updated.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/PolicyRestrictionContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "PolicyRestriction"
+        ],
+        "operationId": "Policy_Update",
+        "description": "Updates the policy restriction configuration of the Api Management service.",
+        "x-ms-examples": {
+          "ApiManagementUpdatePolicyRestriction": {
+            "$ref": "./examples/ApiManagementUpdatePolicyRestriction.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/PolicyRestrictionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./definitions.json#/definitions/PolicyRestrictionUpdateContract"
+            },
+            "description": "The policy restriction to apply."
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/IfMatchRequiredParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Policy restriction configuration of the Api Management service was successfully updated.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/PolicyRestrictionContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version",
+                "type": "string"
+              }
+            }
+          },
+          "201": {
```

Comment:
>201 [](http://example.com/codeflow?start=11&length=3)

PATCH should not return a 201 , only 200 for sync operations : 

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#response-1](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#response-1)